src/Illuminate/Database/Eloquent/Factories/HasFactory.php

```diff
@@ -34,6 +53,10 @@ protected static function newFactory()
             return static::$factory::new();
         }
 
+        if($useFactory = static::getUseFactoryAttribute()) {
+            return $useFactory;
+        }
+
         return null;
```

Comment:
You could probably shorten this to the following 🤔
```suggestion
        return static::getUseFactoryAttribute();
```

or, if, for some reason, you want/need to be explicit:
```suggestion
        return static::getUseFactoryAttribute() ?? null;
```

Reply:
Yeah I went back and forth on this. I originally was just returning the method since it returns null without an attribute but opted for something more clear (albeit a bit redundant). I like the coalescing, that's a good happy medium IMO.